src/sentry/static/sentry/app/actionCreators/events.jsx

```diff
@@ -70,8 +70,8 @@ export const doEventsRequest = (
   api,
   {
     organization,
-    projects,
-    environments,
+    project,
+    environment,
```

Comment:
Does plural better reflect what's being represented here? (multi-project, multi-environment requests)

Reply:
It does, but I wanted to make them consistent with API and url parameters.

Reply:
I realise this may not be possible/easy to change especially if anyone is using that API already, but plural would also make more sense in both of those places you mentioned

Reply:
API changes are not something I'll address in this PR (and I'm going to also followup with changes to `statsPeriod` vs `period`) -- additionally, I think I wanted plural to begin with, but it remained singular since that's generally how our API is (at least that's what grepping around for `requestgetlist` shows).